### PR TITLE
[#5] [UI] As a user, I can see product list in the cart section

### LINKED ISSUE #5: [UI] As a user, I can see product list in the cart section
## Why

When the user clicked on the sign-in button. It should navigate the user to MainScreen and they can see the product list in the cart.

## Acceptance Criteria

- Display the product list in the cart section:
  - Display the title `CART`
  - Display the product item for 3 items:
      - Display the image item.
      - Display the description.
      - Display the prices section.
          Screen Shot 2566-07-03 at 16 29 24
      - Display the `bin` icon button.
  
- The sub-sections `Total`, `Voucher Code`, and `Payment Method` in this story will be left as blank spaces. All of them will be handled in separate tickets.  

## Design

Screen Shot 2566-07-03 at 16 19 52

## Resources

https://www.figma.com/file/QQJEM3O8LmntPcU8UrpXOU/POS-iOS-application?type=design&node-id=338-217&mode=design&t=DXWd1nPcwzyjKp0f-4

### PR DESCRIPTION
**Pull request by kaungkhantsoe**
- Close #5 

## What happened 👀

When the user clicked on the sign-in button. It should navigate the user to MainScreen and they can see the product list in the cart.

## Insight 📝

N/A

## Proof Of Work 📹

Screenshot 2023-07-12 at 3 07 00 PM

### CODE REVIEW

File: lib/screen/main/main_screen.dart [hunk 1]
```diff
@@ -20,61 +21,103 @@ class _MainScreenState extends ConsumerState<MainScreen> {
...
+                ],
+              ),
+            ),
+          ),
+          Expanded(
+            child: Row(
+              children: [
+                const VerticalDivider(
+                  width: 1,
+                  color: ColorName.whitePlatinumColor,
```
Comment (Wadeewee) [inline lib/screen/main/main_screen.dart:78]:
```suggestion
            const VerticalDivider(
              width: 1,
               color: ColorName.whitePlatinumColor,
            ),
             Expanded(
                child: _buildCartSection(),
             )
```

File: lib/screen/main/main_screen.dart [hunk 2]
```diff
@@ -20,61 +21,103 @@ class _MainScreenState extends ConsumerState<MainScreen> {
...
+  Widget _buildCartSection() {
+    return Container(
+      color: Colors.white,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(
+              top: Dimens.space30,
+              left: Dimens.space39,
```
Comment (Wadeewee) [inline lib/screen/main/main_screen.dart:98]: Missing padding for the right?
```suggestion
              left: Dimens.space39,
              right: Dimens.space39,
```

File: lib/screen/main/cart_item.dart [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                            color: Colors.black,
+                          ),
+                        ),
+                      ),
+                      Expanded(child: Container()),
+                      SizedBox(
+                        width: Dimens.space24,
+                        height: Dimens.space24,
+                        child: Image.asset(
+                          Assets.images.trashBin.path,
```
[the remark this answers is not in the document]
Comment (Wadeewee) [reply lib/screen/main/cart_item.dart:80]: It seems the icon for the bin is not aligned with the end by comparing Figma and pow, Could you please recheck 🙏 

- POW

Screen Shot 2566-07-12 at 12 31 03

- Figma design

Screen Shot 2566-07-12 at 12 31 35

**Comment by Wadeewee** [comment]
Rest LGTM 🚀

File: lib/screen/main/main_screen.dart (hunk 2 above)
Comment (kaungkhantsoe) [reply lib/screen/main/main_screen.dart:98]: We don't really need the right padding but I will add it.

**Comment by kaungkhantsoe** [comment]
@Wadeewee All updated 🙏

**Review by Wadeewee — APPROVED** [review]

**Review by Wadeewee — APPROVED** [review]

**Pull request merged**